.github/workflows/release.yml

```diff
@@ -0,0 +1,71 @@
+name: "release"
+
+on: 
+  push:
+    branches:
+      - 'release'
+
+jobs:
+  Spec:
+    runs-on: self-hosted
+    strategy:
+      matrix:
+        check: [parallel_spec, 'syntax lint metadata_lint check:symlinks check:git_ignore check:dot_underscore check:test_file rubocop']
+        ruby_version: [2.5.x]
+        puppet_gem_version: [~> 5.0, ~> 6.0]
+        exclude:
+        - puppet_gem_version: ~> 5.0
+          check: 'syntax lint metadata_lint check:symlinks check:git_ignore check:dot_underscore check:test_file rubocop'
+        - ruby_version: 2.5.x
+          puppet_gem_version: ~> 5.0
+    steps:
+    - uses: actions/checkout@v1
+    - name: Spec Tests
+      uses: puppetlabs/action-litmus_spec@master
+      with:
+        puppet_gem_version: ${{ matrix.puppet_gem_version }}
+        check: ${{ matrix.check }}
+  LitmusAcceptance:
+    env:
+      HONEYCOMB_WRITEKEY: 7f3c63a70eecc61d635917de46bea4e6
+      HONEYCOMB_DATASET: litmus tests
+    runs-on: self-hosted
+    steps:
+    - uses: actions/checkout@v1
+    - name: Build
+      run: |
+        gem install bundler
+        bundle install --path .bundle/gems --jobs 4 --retry 3
+        bundle exec rake spec_prep
+    - name: Prepare inventory file
+      run: |
+        cat <<EOF >> inventory.yaml
+        ---
+        version: 2
+        groups:
+        - name: docker_nodes
+          targets: []
+        - name: ssh_nodes
+          targets: []
+        - name: winrm_nodes
+          targets: []
+        EOF
+    - name: Provision and Setup
+      run: |
+        export LANG=en_US.UTF-8
+        export LANGUAGE=en_US:en
+        eval "$(ssh-agent -s)"
+        ssh-add ~/.ssh/id_rsa-acceptance
+        bundle exec bolt --modulepath spec/fixtures/modules -i ./inventory.yaml plan run satellite_pe_tools::test_01_provision
+        bundle exec bolt --modulepath spec/fixtures/modules -i ./inventory.yaml plan run satellite_pe_tools::test_02_server_setup
+    - name: Inventory
+      run: |
+        cat ./inventory.yaml
+    - name: Run tests
+      run: |
+        env
+        bundle exec bolt --modulepath spec/fixtures/modules -i ./inventory.yaml plan run satellite_pe_tools::test_03_test_run
```

Comment:
@david22swan can we use the integeration tag in rakefile and run the tests using the following command
https://github.com/puppetlabs/puppetlabs-websphere_application_server/blob/master/Rakefile#L94
https://github.com/puppetlabs/puppetlabs-websphere_application_server/blob/master/.github/workflows/weekly.yml#L59

Reply:
Could you please add weekly workflow too. Thank you.

Reply:
Tried the integration tag but it kept throwing an error, `Target 'litmus_localhost' not found in inventory.yaml`.
Think we did something different at some point.
Weekly workflow added.

Reply:
Done